.nvmrc

```diff
@@ -0,0 +1 @@
+12.14.0
```

Comment:
This should be 12.13.1

Reply:
Oops, sorry @pbredenberg. @jthomerson filled me in that we will be upgrading to 12.14.0 now.